internal/parser/simple_parser_rules.go

```diff
@@ -1928,6 +1930,25 @@ func (p *simpleParser) parseResultColumn(r reporter) (stmt *ast.ResultColumn) {
 				p.consumeToken()
 			}
 		}
+	default:
+		stmt.Expr = p.parseExpression(r)
+		next, ok := p.optionalLookahead(r)
```

Comment:
why optional lookahead and check for EOF? why no check for statement separator? why not use lookahead instead?

Reply:
OptionalLookahead is used because there may or may not be a token ahead of this.
Last time I asked which function I must use for such a situation I remember you mentioning optionalLookahead is the way to go.\

Reply:
Ok, true, but you should also check for StatementSeparator then, not only EOF

Reply:
Uh oh. That means the entire file will undergo changes

Reply:
I trust you didn't miss any 😄

Reply:
Done

Reply:
VSCode's search and replace :D